### ISSUE TITLE
Non coding sequence positive selection batch file in HyPhy

### ISSUE THREAD

**Issue by va89fa** [veg/hyphy#998]
Hello, 

I want to use 'testforpositiveselection' batch file in hyphy 2.2.4 for non coding sequence analysis. where should I copy this batch file manually?
I pasted it in template batch files folder, yet I couldn't find it on positiveselection section of hyphy program when running.
Is there any other batch file in hyphy program for non coding selection pressure analysis? perhaps NielsenYang.bf?

Best,
Vafa

**Comment by spond** [comment]
Dear @va89fa,

You can place the file anywhere you'd like, and then simply call HyPhy with the name full path name, like so, for example

```
$HYPHYMP /home/me/scripts/testforpositiveselection.bf
```

Placing the file inside `TemplateBatchFiles` will not make the file automatically appear in analysis menus. 

Finally, I would suggest you use the latest 2.3x (or even 2.5x) version of HyPhy first, and revert to 2.2.4 only if there is an incompatibility with `testforpositiveselection`

Best,
Sergei

**Comment by va89fa** [comment]
Dear @spond pond

Thanks for your advice. Actually I had used hyphy selection analysis on windows/desktop for my previous publication. According to your suggestion for the path name, I have decided to clone hyphy from github, and use command line prompt. 
I have read the instructions on command line issues on hyphy website, is it referred to command prompt on Mac system?
Sorry if my issue is simple. I haven't used command line much before, so I couldn't run hyphy yet. in fact, after opening the command prompt from github desktop, no directory which I use is accepted.
the present directory on command prompt is :

C:\Users\Vafa\Documents\GitHub\hyphy>

what should be written after ">" sign to execute the analyses?
is there something wrong with installation, that my commands could not be executed?
